recipes/pycoalescence/build.sh

```diff
@@ -0,0 +1,11 @@
+cd pycoalescence
+$PYTHON installer.py --cmake-args="-DCMAKE_PREFIX_PATH=${PREFIX} -DCMAKE_INSTALL_PREFIX=${PREFIX}"
+
+rm -rf __pycache__
```

Comment:
@thompsonsed , please review this build script

Reply:
hum... still getting the wrong compiler... Locally it worked. I need to investigate that